lib/FieldType/Mapper/ImageFormMapper.php

```diff
@@ -36,6 +42,16 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
                 'required' => false,
                 'property_path' => 'validatorConfiguration[FileSizeValidator][maxFileSize]',
                 'label' => 'field_definition.ezimage.max_file_size',
+                'constraints' => [
+                    new Range([
+                        'min' => 0,
+                        'max' => $this->maxUploadSize->get(),
```

Comment:
DB data format inconsistency? That's the reason you're not doing like: `'max' => $this->maxUploadSize->get() / (1024 * 1024),`, right?

Reply:
ImageFieldType expects input in bytes not megabytes